test/e2e/apimachinery/aggregator.go

```diff
@@ -161,12 +165,10 @@ func TestSampleAPIServer(f *framework.Framework, image, namespaceName string) {
 	ns := f.Namespace.Name
 	if framework.ProviderIs("gke") {
 		// kubectl create clusterrolebinding user-cluster-admin-binding --clusterrole=cluster-admin --user=user@domain.com
-		framework.BindClusterRole(client.RbacV1beta1(), "cluster-admin", ns,
-			rbacv1beta1.Subject{Kind: rbacv1beta1.ServiceAccountKind, Namespace: ns, Name: "default"})
-		err := framework.WaitForAuthorizationUpdate(client.AuthorizationV1beta1(),
-			serviceaccount.MakeUsername(ns, "default"),
-			"", "get", schema.GroupResource{Group: "storage.k8s.io", Resource: "storageclasses"}, true)
-		framework.ExpectNoError(err, "Failed to update authorization: %v", err)
+		authenticated := rbacv1beta1.Subject{Kind: rbacv1beta1.GroupKind, Name: user.AllAuthenticated}
+		framework.BindClusterRole(client.RbacV1beta1(), "cluster-admin", ns, authenticated)
+		// TODO: cheftako - Get this working with whoami/ClientSubject & WaitForAuthorizationUpdate
+		time.Sleep(10 * time.Second)
```

Comment:
Is it possible to retry something until it starts working?

maybe you could issue a subject access review call to find out?